public/components/App.js

```diff
@@ -0,0 +1,66 @@
+import DateRangePicker from "./DateRangePicker.js";
+import FlightsTable from "./FlightsTable.js";
+import TableHead from "./TableHead.js";
+import TableBody from "./TableBody.js";
+import TableRow from "./TableRow.js";
+import PaginationLinks from "./PaginationLinks.js";
+
+export default {
+    components: {
+        'date-picker': DateRangePicker,
+        'table-head': TableHead,
+        'flight-table': FlightsTable,
+        'table-body': TableBody,
+        'table-row': TableRow,
+        'pagination-links': PaginationLinks
+    },
+    data() {
+        return {
+            flights: [],
+            links: [],
+            startDate: null,
+            endDate: null
+        }
+    },
+    created() {
+        this.fetchFlights('/api/flights', 'id');
+    },
+    methods: {
+        fetchFlights(url, sort) {
+            axios.get(url, { params: { sort, start_date: this.startDate, end_date: this.endDate } })
+                .then(response => {
+                    this.links = response.data.links;
+                    this.flights = response.data.data.data;
+                })
+                .catch(error => console.error(error));
+        },
+        sortFlights(sortType) {
```

Comment:
`sortType` could be directly 'id', 'departure_time_desc' or 'departure_time_asc' so you save all the comparations in there

Reply:
also I think that separating 'asc'/'desc' from the column we are ordering is a wise idea

Reply:
because for every option (id, departure, etc) you create two options (option_asc and option_desc)